swig/python/gdal-utils/scripts/ogr_layer_algebra.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+import sys
+
+from osgeo.gdal import deprecation_warn
+
+# import osgeo_utils.ogrmerge as a convenience to use as a script
```

Comment:
```suggestion
# import osgeo_utils.ogr_layer_algebra as a convenience to use as a script
```

Reply:
added in https://github.com/kssvrk/gdal/commit/6da6052853cf70eec7e4bb703873e12c037f7b07